ui/src/helpers/api.js

```diff
@@ -76,4 +76,12 @@ export function questionsQuery() {
   return request
     .get('/server/questions')
     .set('Content-Type', 'application/json');
+}
+
+export function saveQuestions(record) {
```

Comment:
Good point. I updated it but am wondering whether this has any actual effect on the underlying database code or is it just for readability?